scripts/linux/smokeTestHelper.sh

```diff
@@ -100,6 +224,7 @@ download-artifact-from-pmc-apt()
   # $2 - target output directory
 
   pkgName=${1%=*}
+  sudo apt --fix-broken install 
```

Comment:
Why needed?

Reply:
Because 1ES is doing questionable things like using linux repo of Ubuntu18.04 instead of repo for Ubuntu20.04 in Ubuntu20.04 images. I manually update the underlying apt repo source to point to the focal one. At times, the command is needed prior to apt update.